app/models/job_io_wrapper.rb

```diff
@@ -74,13 +73,13 @@ def file_actor
     Hyrax::Actors::FileActor.new(file_set, relation.to_sym, user)
   end
 
-  # @return [FileNode, FalseClass] the created file node on success, false on failure
+  # @return [Hyrax::FileNode, FalseClass] the created file node on success, false on failure
```

Comment:
There were a number of yardoc param defs that left off Wings namespace.  So it was added as Hyrax namespace since FileNode was moved there.